weave/trace_server/trace_server_interface.py

```diff
@@ -590,6 +590,55 @@ class EnsureProjectExistsRes(BaseModel):
     project_name: str
 
 
+class CostCreateInput(BaseModel):
+    prompt_token_cost: Optional[float] = 0.015
```

Comment:
why have these defaults? Seems like we would want to require the user to provide the cost?